packages/jss/src/DomRenderer.js

```diff
@@ -33,7 +33,9 @@ const memoize = <Value>(fn: () => Value): (() => Value) => {
   }
 }
 
-type GetPropertyValue = (HTMLElementWithStyleMap | CSSStyleRule | CSSKeyframeRule, string) => string
+type CSSRule = HTMLElementWithStyleMap | CSSStyleRule | CSSKeyframeRule
```

Comment:
this is semantically incorrect, because HTMLElementWithStyleMap is not a CSS Rule, that's why it wasn't done this way

Reply:
I think trying to call it something is not worth the effort

Reply:
Yes this is a mistake, I will to close this PR